src/misc/lv_anim_timeline.c

```diff
@@ -170,21 +180,88 @@ static void anim_timeline_set_act_time(lv_anim_timeline_t * at, uint32_t act_tim
 
         uint32_t start_time = at->anim_dsc[i].start_time;
         int32_t value = 0;
+
```

Comment:
Just a trick to avoid so much `at->anim_dsc[i]`;
Define at beginning `lv_anim_timeline_dsc_t * dsc = &at->anim_dsc[i];` 
Use dsc directly in the for loop.